HISTORY.md

```diff
@@ -14,6 +14,7 @@
 * Smoke Test, remove nullable structs from global namespace. [#2078](https://github.com/TileDB-Inc/TileDB/pull/2078)
 
 ## Improvements
+* Add config option  to toggle bounding read ranges to domain or erroring [#2162](https://github.com/TileDB-Inc/TileDB/pull/2162)
```

Comment:
Looks like an extra space snuck in before 'to toggle'

Reply:
Looks like the action for history didn't handle the backticks around the config option (`sm.read_range_oob_error`)